src/test/java/org/commcare/backend/session/test/BasicSessionNavigationTests.java

```diff
@@ -29,119 +29,119 @@ public void setUp() throws Exception {
     @Test
```

Comment:
Changes in this file are just because I realized that we had the order of arguments to `assertEquals()` backwards; it should be `(expected, actual)`